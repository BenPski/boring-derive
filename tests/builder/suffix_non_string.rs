use boring_derive::Builder;

#[derive(Debug, Default, Builder)]
#[builder(suffix = 1)]
struct Example {
    item: String,
}

fn main() {
    let ex = Example::default();
    println!("{:?}", ex);
}
