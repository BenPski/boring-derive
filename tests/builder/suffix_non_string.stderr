error: suffix must be a string, got `1`
 --> tests/builder/suffix_non_string.rs:4:11
  |
4 | #[builder(suffix = 1)]
  |           ^^^^^^^^^^
